spec/lib/conjur/conjur_config_spec.rb

```diff
@@ -6,8 +6,8 @@
   end
 
   it "reports the attribute source as :defaults" do
-    expect(Conjur::ConjurConfig.new.attribute_sources[:trusted_proxies]).
-      to eq(:defaults)
+    expect(Conjur::ConjurConfig.new.attribute_sources[:trusted_proxies])
+      .to eq(:defaults)
```

Comment:
I think we use trailing dots more often than leading so can we change these back?

Reply:
Dissenting vote :)

Consider:

```rb
        Resource
          .where(identifier.like("#{AUTHN_RESOURCE_PREFIX}%"))
          .where(kind => "webservice")
          .select_map(identifier)
          .map { |id| id[%r{^conjur/(authn(?:-[^/]+)?(?:/[^/]+)?)$}, 1] } # filter out nested status webservice
          .compact
          .push(::Authentication::Common.default_authenticator_name)
```
leading dot does two things:
1. Makes clear at a glance that we're dealing with a continuation (otherwise, if you don't notice the dot at the end of the previous line, it could be a new method call on `self`)
2. Makes cut and paste more ergonomic (otherwise the final item is a special case).